### PR TITLE
CLOUDP-252392: unify go imports installation

### PR DESCRIPTION
**Pull request by wtrocki**
## Description

CLOUDP-252392

Reused existing make file and added documentation for goimports binary that is now integral part of the SDK.

## Verification

- Successfully run [autoupdate-preview job from](https://github.com/mongodb/atlas-sdk-go/actions/runs/9663629705) branch.
- Successfully run `make openapi-pipeline` in evergreen

## Documentation

Added documentation in order to install required tool on the developers machines

## Alternatives considered

- Initially implemented goimports installation during generate phase however this approach would have hidden this tool from developers can cause futher issues with local dev setup.
- Investigated embedding all binaries into go.mod with ability to install them:
```
tools=...
go install $tools
```

### CODE REVIEW

File: tools/scripts/generate.sh [hunk 1]
```diff
@@ -36,4 +36,11 @@ npm exec openapi-generator-cli -- generate \
     --type-mappings=file=io.ReadCloser \
     --ignore-file-override=config/.go-ignore
 
+# Check if goimports exists and is executable
+if ! command -v goimports &> /dev/null; then
+  echo "Error: goimports command not found. Please install goimports by running 'make install-goimports' before running this script."
+  exit 1
+fi
+
```
Comment (wtrocki) [inline tools/scripts/generate.sh:44]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -57,7 +57,9 @@ Reviewers, please ensure that the CLA has been signed by referring to [the contr
 
 #### Prerequisite Tools
 - [Git](https://git-scm.com/)
-- [Go (at least Go 1.14)](https://golang.org/dl/)
+- [Golang (at least Go 1.21)](https://golang.org/dl/)
+    - Repository uses https://asdf-vm.com for golang version management
```
Comment (yelizhenden-mdb) [inline CONTRIBUTING.md:61]:
```suggestion
    - Repository uses [asdf](https://asdf-vm.com) for golang version management
```

Comment (wtrocki) [reply CONTRIBUTING.md:61]: Nice find. Thank you

**Review by yelizhenden-mdb — APPROVED** [review]

**Review by jeroenvervaeke — APPROVED** [review]
LGTM

**Comment by wtrocki** [comment]
Thank you!

**Pull request merged**